docs/Install.md

```diff
@@ -6,22 +6,11 @@ Inductiva package is simple to install, just run on your terminal:
 pip install --upgrade inductiva
 ```
 
-This will provide the core functionalities of the API, which allows you to submit jobs, control machines and run simulations. To use the visualization and post-processing tools, you need to install additional optional dependencies specific to different scientific domains: `molecules_extra`, `fluids_extra` or `coastal_extra`. For example, for fluid dynamics:
-
-```
-pip install --upgrade "inductiva[fluids_extra]"
-```
-
+This will provide the core functionalities of the API, which allows you to submit jobs, control machines and run simulations. 
```

Comment:
80 character line length

Reply:
@hpenedones @sergiomsantos check out the final iteration on this pull request so we merge it today.
- fixed 80 characters per line on this PR to avoid conflicts
- made edits on the wording
- removed unnecessary text

For the future, we might consider expanding on this section to include an FAQ, some requirements before installing inductiva (latest updates, etc...)